### ISSUE TITLE
Failure adding share with slash in name

### ISSUE THREAD

**Issue by permezel** [MobiusSync/MobiusSync#66]
I added a share and named it "dap/xxx" which works fine on linux. Got this error when I tried to accept it on iOS. ![image](https://user-images.githubusercontent.com/2313876/151914555-9751db07-95a8-43e2-bf4d-4fbc59373d84.png)

**Comment by SimonPickup** [comment]
Just a guess, is the problem that you have a / in the folder name?

**Comment by permezel** [comment]
Indeed. I intentionally put the / in because I want the shared folders to be hierarchically named and i want to use native file system hierarchy on linux and macos. Works fine except for möbius. Möbius will not let me edit the local path to fix things, even if i create the hierarchy by hand.

**Comment by permezel** [comment]
Just checked the macos one.  Seems the local path became "~/dap CryptoMatorTest".
On linux, it became "/home/user/dap/CryptoMatorTest" which I was expecting. ($HOME is /home/user here).

I created it on my synology NAS and named the share "dap/CryptoMatorTest" and I have auto-share enabled.

**Comment by SimonPickup** [comment]
Do you find the MacOS behaviour acceptable?

**Comment by permezel** [comment]
> Do you find the MacOS behaviour acceptable?

Where possible, I avoid space in file names. Gratuitous blanks, No thanx.

**Comment by rudle** [comment]
I’m having the same issue. I’m trying to sync an existing directory onto iOS. It’s not clear to me where the slash is coming from but Mobius is reporting the same error as the OP.

```
2022-04-17 10:23:13: Failed to create folder root directory mkdir /private/var/mobile/Containers/Data/Application/<UUID>/org: operation not permitted

2022-04-17 10:23:13: Error on folder "org" (<folder-id>) at ~/org: folder path missing
```

A snippet from my my config.xml

```xml
 <folder id="<folder-id>" label="org" path="<absolute-path>" type="sendreceive" rescanIntervalS="3600" fsWatcherEnabled="true" fsWatcherDelayS="10" ignorePerms="false" autoNormalize="true">
…
       <device id="DEVICE-ID" introducedBy="">
          <encryptionPassword></encryptionPassword>
        </device>
…
</folder>
```

I agree with the above comment that allowing the user to edit the local path by hand might remedy this issue.
